packages/common/src/store/buy-usdc/sagas.ts

```diff
@@ -0,0 +1,215 @@
+import { PublicKey } from '@solana/web3.js'
+import BN from 'bn.js'
+import { takeLatest } from 'redux-saga/effects'
+import { call, put, race, select, take } from 'typed-redux-saga'
+
+import { Name } from 'models/Analytics'
+import { ErrorLevel } from 'models/ErrorReporting'
+import {
+  getTokenAccountInfo,
+  pollForBalanceChange
+} from 'services/audius-backend/solana'
+import { IntKeys } from 'services/remote-config'
+import { getContext } from 'store/effects'
+import { setVisibility } from 'store/ui/modals/slice'
+import { initializeStripeModal } from 'store/ui/stripe-modal/slice'
+
+import { getBuyUSDCProvider } from './selectors'
+import {
+  buyUSDCFlowFailed,
+  buyUSDCFlowSucceeded,
+  onRampCanceled,
+  onRampOpened,
+  onRampSucceeded,
+  startBuyUSDCFlow
+} from './slice'
+import { USDCOnRampProvider } from './types'
+import { getUSDCUserBank } from './utils'
+
+// TODO: Configurable min/max usdc purchase amounts?
+function* getBuyUSDCRemoteConfig() {
+  const remoteConfigInstance = yield* getContext('remoteConfigInstance')
+  yield* call([remoteConfigInstance, remoteConfigInstance.waitForRemoteConfig])
+  const retryDelayMs =
+    remoteConfigInstance.getRemoteVar(IntKeys.BUY_AUDIO_WALLET_POLL_DELAY_MS) ??
```

Comment:
nit: This is the` BUY_AUDIO` poll delay ms, which might be confusing in the future. I think it's totally fine to reuse though

Reply:
Ah yep. I can probably rename it to a generic constant. The idea was to use the same behavior we've done for buying audio. But this _is_ confusing.